server/tests/mcp/test_mcp_registry.py

```
@@ -37,6 +40,155 @@ def test_search_dispatch_entries_query_filter():
+    }
+    for name, skill in expected.items():
+        assert name in by_name, f"{name} missing from allowlist"
+        entry = by_name[name]
+        assert entry.enabling_skills == (skill,), name
+        assert entry.method == "GET", name
+
+
+def test_new_connector_entries_gated_in_search(monkeypatch):
+    """Gated entries are only callable_now once their skill is connected."""
+    monkeypatch.setattr(
+        registry, "_check_skill_connected", lambda s, u: s == "sentry",
```

Ruff (0.15.15)

[warning] Unused lambda argument: `u`

(ARG005)